ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -356,8 +354,11 @@ Launch And Access Workbench
 Open Workbench
     [Documentation]    Clicks on "open" link for a given workbench
     [Arguments]    ${workbench_title}
-    Click Link       ${WORKBENCH_SECTION_XP}//tr[td[@data-label="Name"]/*[div[text()="${workbench_title}"]]]/td//a[text()="Open"]
-    Switch Window   NEW
+    ${open_workbench_link}=    Set Variable
+    ...    ${WORKBENCH_SECTION_XP}//tr[td[@data-label="Name"]/*[div[text()="${workbench_title}"]]]/td//a[text()="Open"]
+    Wait Until Page Contains Element    ${open_workbench_link}    timeout=30s
```

Comment:
this may hide some prod issue like workbench Running but not reporting the link to open. This was something captured in PSAP test runs

Reply:
I was referring to issues like https://issues.redhat.com/browse/RHODS-5819 where the link wasn't ready after the status of workbench was Running. So i'm wondering if adding this additional ait may hide similar issues